hw_28_students/js/script.js

```diff
@@ -0,0 +1,200 @@
+// Список студентів та система оцінювання
+
+const students = [
+    {
+        fName: 'Bill',
+        lName: 'Clinton',
+        age: 20,
+        marks: [3, 3, 5, 8, 4]
+    },
+    {
+        fName: 'Will',
+        lName: 'Smith',
+        age: 19,
+        marks: [10, 3, 7, 8, 1]
+    },
+    {
+        fName: 'Angelina',
+        lName: 'Jolie',
+        age: 24,
+        marks: [6, 6, 5, 10, 3]
+    },
+    {
+        fName: 'Tina',
+        lName: 'Turner',
+        age: 17,
+        marks: [10, 9, 9, 4, 9]
+    },
+    {
+        fName: 'Laura',
+        lName: 'Palmer',
+        age: 22,
+        marks: [5, 7, 3, 10, 4]
+    },
+    {
+        fName: 'Jacky',
+        lName: 'Chan',
+        age: 26,
+        marks: [6, 8, 5, 10, 1]
+    },
+    {
+        fName: 'Timothy',
+        lName: 'Chalamet',
+        age: 18,
+        marks: [8, 10, 5, 9, 9]
+    },
+    {
+        fName: 'Lea',
+        lName: 'Seydoux',
+        age: 23,
+        marks: [10, 5, 7, 7, 1]
+    },
+]
```

Comment:
why do you use fName, lName arguments for construstor? In task description were name, surname